### PR TITLE
Update JCM Adapter to 1.0 

### PR DESCRIPTION
**Pull request by JCarterGw**
## Type of change

-- [ ] Other

## Description of change

Update JCM adapter for 1.0 compatability

- test parameters for validating bids
```
var adUnits = [
  {
    code: 'test-div1',
    sizes: [[300, 250]], // display 300x250 
    bids: [
      {
      bidder: 'jcm',
      params: {
      siteId: '3608'
     }
  }
 ]
},{
  code: 'test-div2',
  sizes: [[728, 90]], // display 728x90
  bids: [
    {
    bidder: 'jcm',
    params: {
      siteId: '3608'
     }
   }
  ]
 }
];

- contact email of the adapter’s maintainer
george@jcartermarketing.com
- [ ] official adapter submission

### CODE REVIEW

File: adapters.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
+[
```
Comment (dbemiller) [inline adapters.json]: File committed accidentally?

File: package.json [hunk 2]
```diff
@@ -11,13 +11,15 @@
     "type": "git",
     "url": "https://github.com/prebid/Prebid.js.git"
   },
+
```
Comment (dbemiller) [inline package.json]: More unintended changes?

File: test/spec/modules/jcmBidAdapter_spec.js [hunk 3]
```diff
@@ -1,244 +1,123 @@
...
-      expect(bidObjArr).to.have.property('bids');
-      var bidObj = bidObjArr.bids[0];
+    ];
 
-      expect(bidObj).to.have.property('adSizes').and.to.equal('300x250');
-      expect(bidObj).to.have.property('siteId').and.to.equal('3608');
-      expect(bidObj).to.have.property('callbackId').and.to.equal('3c9408cdbf2f68');
+    it('sends bid request to ENDPOINT via GET', () => {
+      const request = spec.buildRequests(bidRequests);
+      expect(request.method).to.equal('GET');
```
Comment (dbemiller) [inline test/spec/modules/jcmBidAdapter_spec.js]: Needs some real assertions... barely tests `buildRequests` at all.

**Comment by dbemiller** [comment]
screen shot 2017-10-18 at 10 11 48 am

File: package.json (hunk 2 above)
Comment (JCarterGw) [reply package.json]: There should be no changes to package.json or adapters.json in this PR.

File: test/spec/modules/jcmBidAdapter_spec.js (hunk 3 above)
Comment (JCarterGw) [reply test/spec/modules/jcmBidAdapter_spec.js]: Will look into this.

File: package.json (hunk 2 above)
Comment (dbemiller) [reply package.json]: ok. Please remove them, then

**Comment by dbemiller** [comment]
nope... still getting the same error

**Comment by JCarterGw** [comment]
Please retest the ad request.  The cross-origin issue is resolved.

**Pull request merged**